domain/schema/controller/sql/0009-controller-node.sql

```diff
@@ -23,3 +23,20 @@ CREATE TABLE controller_node_agent_version (
     FOREIGN KEY (architecture_id)
     REFERENCES architecture (id)
 );
+
+CREATE TABLE controller_api_address (
+    controller_id TEXT NOT NULL,
+    -- The value of the configured IP address.
+    -- e.g. 192.168.1.2 or 2001:db8:0000:0000:0000:0000:0000:00001.
+    address_value TEXT NOT NULL,
+    -- one of ipv4, ipv6 etc.
+    type_id INT NOT NULL,
+    -- one of public, local-cloud, local-machine, link-local etc.
+    scope_id INT NOT NULL,
+    port INT NOT NULL,
+    space_uuid TEXT NOT NULL,
```

Comment:
FK?

Reply:
This one you can't, since it's not the same db. It's the watcher that's going to update it looking for the values in the controller model db.

Reply:
We discussed this. The table is generated from the application to the addresses of:
- Scope matching.
- Space filtering.
- API port appending.

There is only one `TEXT` address column in addition to the controller ID.